src/Microdown-RichTextComposer/MicRichTextComposer.class.st

```diff
@@ -44,8 +44,21 @@ Class {
 	#superclass : #MicrodownVisitor,
 	#instVars : [
 		'canvas',
-		'codeStyler',
-		'textStyler'
+		'monospaceBackgroundColor',
+		'superscriptColor',
+		'subscriptColor',
```

Comment:
Reintroducing these instance variables in MicRichTextComposer is wrong, they were moved to the TextStyler in one of the earlier pull requests I made. Perhaps you forgot to merge those into your base before pushing?